WooCommerce/Classes/POS/Presentation/PointOfSaleDashboardView.swift

```diff
@@ -59,6 +59,37 @@ struct PointOfSaleDashboardView: View {
                 }
             }
         })
+        .posModal(isPresented: $viewModel.showExitPOSModal, fixedWidth: false, fixedHeight: false, content: {
+            VStack(spacing: 0 ) {
+                HStack {
+                    Spacer()
+                    Button {
+                        viewModel.showExitPOSModal = false
+                    } label: {
+                        Image(PointOfSaleAssets.xClose.imageName)
+                            .frame(width: Constants.closeIconSize, height: Constants.closeIconSize)
+                            .foregroundColor(Color.posTertiaryTexti3)
```

Comment:
You could consider putting all this in a
`.overlay(alignment: .topTrailing)` modifier – see `FeatureAnnouncementCardView` for an example